config.toml

```diff
@@ -138,11 +138,11 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.20"
+latest = "v1.21"
 
-fullversion = "v1.20.0"
-version = "v1.20"
-githubbranch = "master"
+fullversion = "v1.21.0"
+version = "v1.21"
+githubbranch = "v1.21.0"
```

Comment:
I don't think we want to make a change to the `githubbranch` and keep it as `master`

Reply:
I'll make the change